src/qtiCommonRenderer/renderers/interactions/InlineChoiceInteraction.js

```diff
@@ -63,11 +63,19 @@ var render = function(interaction, options) {
         $container.find('option[value=' + _emptyValue + ']').remove();
     }
 
+    const getItemDir = () => {
+        let item = $container.closest('.qti-item');
+        const itemBody = item.find('.qti-itemBody');
+        const itemDir = itemBody.attr('dir');
+        return itemDir;
```

Comment:
niptick: add a default value to `ltr` just to be safe (in case we have an old item for example)